portal/src/main/webapp/WEB-INF/jsp/mutation_details.jsp

```diff
@@ -76,6 +78,23 @@
         var sampleArray = window.QuerySession.getSampleIds();
         OncoKB.setUrl('<%=oncokbUrl%>');
         var mutationProxy = DataProxyFactory.getDefaultMutationDataProxy();
+        var annotationCol = null;
+
+        if(OncoKB.getAccess()) {
+            var oncokbInstanceManager = new OncoKB.addInstanceManager();
```

Comment:
Totally. I'm using this in patient view as well. Any plan to move to patient view?